### PR TITLE
Pull request for Avatar component

### PR DESCRIPTION
**Pull request by SanjayJatti**
avatar

### CODE REVIEW

**Comment by omgaikwad** [comment]
The code and avatar.css looks fine, I think you are good to go!

File: Componets/Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.avatar-container{
+    margin: 2rem;;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+}
+.avatar{
+    border-radius: 50%;
+    border: 1x solid var(--secondary-color)
```
Comment (omgaikwad) [inline Componets/Avatar/avatar.css]: You mistyped "1x" instead of "1px", try changing this.

**Review by omgaikwad — CHANGES_REQUESTED** [review]

**Review by vijaytembugade — APPROVED** [review]
Looks clean and nice. Good to go

File: Layout/Sidebar/sidebar.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.sidebar{
+    position: fixed;
+    min-height: 88vh;
+    top:12vh;
+}
+.sidebar-ul{
```
Comment (prachi-sahani) [inline Layout/Sidebar/sidebar.css]: can be named as "sidebar-list"

**Review by prachi-sahani — COMMENTED** [review]
Generic feedback:
- naming convention can improved
- try to use consistent units(even for colors)

**Comment by t007rushi** [comment]
Looks Good One suggestion is you can show different sizes as separate section & border-colored different as your get clarity

File: Componets/Avatar/avatar.css (hunk 1 above)
Comment (SanjayJatti) [reply Componets/Avatar/avatar.css]: Typing mistake...corrected. Thanks for feedback.

File: Layout/Sidebar/sidebar.css (hunk 2 above)
Comment (SanjayJatti) [reply Layout/Sidebar/sidebar.css]: Yup...rectified with better naming conventions. Thanks for valuable `feedback`
`.sidebar-list{
    display: flex;
    flex-direction: column;
    justify-content: space-around;
    height: 88vh;
    width: 20vw;
    border-right: 1px solid var(--light-secondary-color);
    text-align: center;
    background: white;
}`

File: Componets/Avatar/avatar.css (hunk 1 above)
Comment (SanjayJatti) [reply Componets/Avatar/avatar.css]: `.avatar{
    border-radius: 50%;
    border: 1x solid var(--secondary-color)
}`

**Pull request merged**